contents.po

```diff
@@ -16,4 +16,4 @@ msgstr ""
 
 #: contents.rst:3
 msgid "Python Documentation contents"
-msgstr "Contenu de la documentation Python"
+msgstr "Contenu de la documentation Python ?"
```

Comment:
Il y a un espace insécable en trop ici :



```suggestion
msgstr "Contenu de la documentation Python ?"
```

---
<details>
  <summary>Disclaimer</summary>

Je suis un robot fait par l'équipe de [l'AFPy et de Traduction](https://github.com/AFPy/PyDocTeur/graphs/contributors)
sur leur temps libre. Je risque de dire des bétises. Ne me blâmez pas, blamez les développeurs.

[Code source](https://github.com/afpy/pydocteur)

I'm a bot made by the [Translation and AFPy teams](https://github.com/AFPy/PyDocTeur/graphs/contributors) on their free
time. I might say or do dumb things sometimes. Don't blame me, blame the developer !

[Source code](https://github.com/afpy/pydocteur)

`PyDocTeur v1.9.3`

</details>